pandas/tests/io/formats/test_to_string.py

```diff
@@ -0,0 +1,222 @@
+from datetime import datetime
+from io import StringIO
+
+import numpy as np
+import pytest
+
+from pandas import DataFrame, Series, option_context, to_datetime
+
+
+def test_repr_embedded_ndarray():
+    arr = np.empty(10, dtype=[("err", object)])
+    for i in range(len(arr)):
+        arr["err"][i] = np.random.randn(i)
+
+    df = DataFrame(arr)
+    repr(df["err"])
+    repr(df)
+    df.to_string()
```

Comment:
Are we missing an assertion here (same below)?

Reply:
sure, but do as a followup